src/client.rs

```diff
@@ -1593,18 +1570,27 @@ impl Spotify {
     ///
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-recommendations)
     #[maybe_async]
-    pub async fn recommendations<L: Into<Option<u32>>>(
+    pub async fn recommendations(
         &self,
+        payload: &Map<String, Value>,
         seed_artists: Option<Vec<&ArtistId>>,
-        seed_genres: Option<Vec<String>>,
+        seed_genres: Option<Vec<&str>>,
         seed_tracks: Option<Vec<&TrackId>>,
-        limit: L,
-        market: Option<Market>,
-        payload: &Map<String, Value>,
+        limit: Option<u32>,
```

Comment:
The `limit` parameter should be the last parameter as what `CHANGELOG.md` claims.

Reply:
good catch